src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/ApartmentValueFunction.java

```diff
@@ -1,180 +1,153 @@
-	 *                    calculation of the presence of a television subjective
-	 *                    values are based.
+	 * An object on which the calculation of the presence of a television subjective
+	 * values are based.
 	 */
 	private PartialValueFunction<Boolean> teleValueFunction;
 
 	/**
-	 * @floorAreaSubjectiveValueWeight The weight associated to the floor area
-	 *                                 subjective value in the calculation of the
-	 *                                 Apartment total subjective value
+	 * The weight associated to the floor area subjective value in the calculation
+	 * of the Apartment total subjective value
 	 */
 	private double floorAreaSubjectiveValueWeight;
 
 	/**
-	 * @nbBedroomsSubjectiveValueWeight The weight associated to the number of
-	 *                                  bedrooms subjective value in the calculation
-	 *                                  of the Apartment total subjective value
+	 * The weight associated to the number of bedrooms subjective value in the
+	 * calculation of the Apartment total subjective value
 	 */
 	private double nbBedroomsSubjectiveValueWeight;
 
 	/**
-	 * @nbSleepingSubjectiveValueWeight The weight associated to the accommodation
-	 *                                  capacity subjective value in the calculation
-	 *                                  of the Apartment total subjective value
+	 * The weight associated to the accommodation capacity subjective value in the
+	 * calculation of the Apartment total subjective value
 	 */
 	private double nbSleepingSubjectiveValueWeight;
 
 	/**
-	 * @nbBathroomsSubjectiveValueWeight The weight associated to the number of
-	 *                                   bathrooms subjective value in the
-	 *                                   calculation of the Apartment total
-	 *                                   subjective value
+	 * The weight associated to the number of bathrooms subjective value in the
+	 * calculation of the Apartment total subjective value
 	 */
 	private double nbBathroomsSubjectiveValueWeight;
 
 	/**
-	 * @terraceSubjectiveValueWeight The weight associated to the presence of a
-	 *                               terrace subjective value in the calculation of
-	 *                               the Apartment total subjective value
+	 * The weight associated to the presence of a terrace subjective value in the
+	 * calculation of the Apartment total subjective value
 	 */
 	private double terraceSubjectiveValueWeight;
 
 	/**
-	 * @floorAreaTerraceSubjectiveValueWeight The weight associated to the floor
-	 *                                        area of an existing terrace subjective
-	 *                                        value in the calculation of the
-	 *                                        Apartment total subjective value
+	 * The weight associated to the floor area of an existing terrace subjective
+	 * value in the calculation of the Apartment total subjective value
```

Comment:
Cf. discussion sur autre PR : la multiplication de ces commentaires pollue plutôt qu’informe.

Reply:
Simplifying the comments on commit 55ad0556d6daa53f539931bf224c3d86846e3ad7